tests/plugins/font/customfontdefinition.js

```diff
@@ -96,17 +125,17 @@
 		},
 
 		'test setdata with font tag with size': function() {
```

Comment:
This test stopped working on IE9 (and only there) with an error
```
Unexpected error: Unable to get value of the property 'html': object is null or undefined
Expected: undefined (undefined)
Actual:   undefined (undefined)
```

So it doesn't seem connected with a fix, but rather with tests refactoring - but nothing important really changed 🤔  Could you please verify if it fails for you too?